src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -23,9 +23,10 @@ contributors:
     sharang,
     chrisdavidmills,
     noraj,
+    akhilpanchal
   ]
-last_updated_by: noraj
-date: 2020-10-11 18:11:47
+last_updated_by: akhilpanchal
+date: 2020-11-15 12:46:15
```

Comment:
Nit, we may update the date before merging (not strictly mandatory to be fair)

Reply:
Hmm -- I'll go ahead and update it to reflect the next deploy date of Dec 03.